.github/workflows/ci.yaml

```diff
@@ -280,7 +280,7 @@ jobs:
           CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
 
   RaspberryPi:
-    if: github.repository == 'ultralytics/ultralytics' && (github.event_name == 'schedule' || github.event.inputs.raspberrypi == 'true')
```

Comment:
@lakshanthad if everything's good should we uncomment this line?

Reply:
correct! I left it for you to review, uncomment and merge! @glenn-jocher